### DISCUSSION TITLE
패키지 빌드 용량

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#311] [5. Q&A]
안녕하세요

빌드 패키지 용량 관련해서 문의하고 싶은데요

최초 패키지 용량 350mb로 제작 후 텍스처가 들어간 재질 또는 오브젝트는 추가하지 않고,

박스 콜라이더 생성을 위해 큐브 오브젝트만 여러개 생성하여 다시 리빌드를 해 보니 550mb로 200mb가 들어났습니다.

다시 추가한 큐브 오브젝트를 인스펙터에서 제거하고 빌드를 해 봐도 용량은 줄어들지 않았습니다. 

어떤 파일이 용량을 차지 하고 있는지 확인을 하고 싶은데 Open Editor Log에도 패키지 구성 항목이 나오지 않아

어떻게 확인을 해야 하는지 찾지 못하고 있는 상황입니다. 어떻게 확인을 하면 될까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

world package 1.1.3 버전 이상부터 패키지에 포함되는 리소스 Log 확인이 가능합니다.

$WORKSPACE/Build/ZEPETO.Script/{worldId}_{version}_{date}.json 파일에서 어떤 리소스가 포함되어 있는지 확인 부탁드립니다.

**Reply by LeeYongSu1** [reply]
감사합니다~
